docs/wiki/user/ground-spike-antenna.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Ground Spike Antenna
+---
+
+The GSA is a radio antenna which can be deployed in the field which allows for any radio to be connected. Doing so will increase the transmit power to 20 W and improve reception due to the increased height of the antenna, generally resulting in an improved quality over range.
```

Comment:
Not all radios can be connected.  Additionally I would write: "improve the quality of the transmission due to the increased height" instead of "improve reception due to the increased"

Reply:
I believe this is the GSA antenna definition: https://github.com/IDI-Systems/acre2/blob/0d2496df9424c7b25c4c981e58982d964ccfbbac/addons/sys_antenna/CfgAcreComponents.hpp#L134

Looks like all radios currently in ACRE **except the 343** can be connected. I'd just list them all below the paragraph.